src/test/java/com/example/auctionapp/AuctionAppApplicationTests.java

```diff
@@ -1,13 +1,130 @@
 package com.example.auctionapp;
```

Comment:
Tests for different classes should reflect the package for the class that is being tested.
If you are testing `com.example.auctionapp.service.AuthenticationService`,
then the test folder equivalent should be placed `com.example.auctionapp.service`.
Different tests should go in different test classes